AirCasting/SessionViews/SessionHeaderView.swift

```diff
@@ -132,6 +159,11 @@ struct SessionHeaderView: View {
             return nil
         }
     }
+    
+    func lastMicMeasurement() -> Int {
+        return Int(session.dbStream?.latestValue ?? 0)
```

Comment:
Can the value be negative? Wouldn't a UInt be a better return type?

Reply:
For now it's only negative